nexus_constructor/field_widget.py

```diff
@@ -253,7 +250,9 @@ def value(self) -> Union[Dataset, Group, Link, None]:
         elif self.field_type == FieldType.kafka_stream:
             return_object = self.streams_widget.get_stream_group()
         elif self.field_type == FieldType.link:
-            return_object = Link(name=self.name, target=self.value_line_edit.text())
+            return_object = Link(
+                parent_node=None, name=self.name, source=self.value_line_edit.text()
```

Comment:
`parent_node` should be set somewhere. Where?

Reply:
Yes, however, at the moment this is primarily related to the UI. So either we need to create a new ticket or check if it is addressed by Jonas already. Resolving for now.